### DISCUSSION TITLE
同样的代码，在QTC上完成的，却只能在VS上跑

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#194] [Q&A] [answered]
如题，在QTC中运行代码时会有如下编译错误，导致无法运行
![LUO3YZ5AU)WREI8(CTC}56J](https://user-images.githubusercontent.com/100075314/167586545-da848562-d8dd-4ba5-8ec9-a0c03768d132.png)
但在VS下可正常编译，这可能是什么问题导致的呢，我花费大量的精力试图解决但没有结果，希望得到各位的帮助，这对我很重要，谢谢！

**Comment by RicoloveFeng** [comment, marked as answer]
https://stackoverflow.com/questions/14170770/unresolved-external-symbol-public-virtual-struct-qmetaobject-const-thiscal
